Enrichment sample doesn't build on JDK11

```
[ERROR] /Users/can/src/hazelcast-jet-code-samples/enrichment/target/generated-sources/protobuf/grpc-java/grpc/EnrichmentServiceGrpc.java:[20,18] cannot find symbol
[ERROR]   symbol:   class Generated
[ERROR]   location: package javax.annotation
[ERROR] 
```

--- a/enrichment/pom.xml
+++ b/enrichment/pom.xml
@@ -49,6 +49,11 @@
             <artifactId>grpc-stub</artifactId>
             <version>1.18.0</version>
         </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.3.2</version>
+        </dependency>
     </dependencies>
 
     <build>

